test/unit/uk/gov/hmrc/individualsincomeapi/domain/v2/SaFurtherDetailsSpec.scala

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.uk.gov.hmrc.individualsincomeapi.domain.v2
+
+import org.scalatest.{Matchers, WordSpec}
+import play.api.libs.json.Json
+import uk.gov.hmrc.individualsincomeapi.domain.v2.SaFurtherDetails
+import utils.IncomeSaHelpers
+
+class SaFurtherDetailsSpec extends WordSpec with Matchers with IncomeSaHelpers {
+
+  val ifSa = Seq(createValidSaTaxYearEntry())
+  val ifSaNoData = Seq(createValidSaTaxYearEntryNoDataContainers())
+  val ifSaNoValues = Seq(createValidSaTaxYearEntryNoValues())
+
+  // TODO Q. Do we need the items in this json response to default to zero as other SA responses do?
```

Comment:
TODO

Reply:
I want to leave this question open until I have discussed with @Bongowhack on his return. 👍